config/settings.yml

```diff
@@ -3,7 +3,10 @@
 # Information in this file will be PUBLIC (this file is tracked in git)
 # Declarations in config/settings.local.yml override declarations in this file
 
+email: info@dp.la
```

Comment:
Since this will be `mailer_sender`, e.g. in [`config/initializers/devise.rb`](https://github.com/dpla/primary-source-sets/blob/feature/authorization/config/initializers/devise.rb#L13), could it be named as such here? I could picture reading this without knowing how it's used, and thinking this is a recipient address.  It's also used specifically as a Devise-related setting, so maybe it should be renamed or categorized to make that clear (put under a `devise` or `authentication` element, for instance).

Also, if it's changed to something `@example.org`, it'll be a little safer to have in here as a default value.  (It's supposed to be overridden in `settings.local.yml`, as the comment suggests.